NEWS.rst

```diff
@@ -1,6 +1,36 @@
 NEWS
 ====
 
+20.6.0 - 2020-06-??
+-------------------
+ * This is the last release of Klein expected to support Python 2.
+ * Python 3.8 is now supported by Klein. [`#303 <https://github.com/twisted/klein/pull/303>`_]
+ * Python 3.4 is no longer supported by Klein. [`#284 <https://github.com/twisted/klein/pull/284>`_]
+ * ``klein.app.subroute`` is now also available as ``klein.subroute``. [`#293 <https://github.com/twisted/klein/pull/293>`_]
+ * Support for forms and sessions. [`#276 <https://github.com/twisted/klein/pull/276>`_]
```

Comment:
@glyph might want to describe this in some more useful way?

Reply:
I'm pretty sure that was in 19.6?

"New "forms" and "sessions" subsystems provide official support for POST requests, including CSRF protection, form generation to include CSRF tokens, dependency injection to populate parameters from both the request and session, as well as lightweight JSON API support."

Do you want more than that?

Reply:
It does appear to be in 19.6.  I'll note it there.